NossaCasaDoCodigo.Biblioteca/DAO/LivrosDAO.cs

```diff
@@ -24,17 +25,11 @@ public static string Salvar(Livro novoLivro)
             return $"O livro {novoLivro.Titulo} foi cadastrado com sucesso!";
         }
 
-        public static List<Livro> Buscar(string busca)
+        public static List<Livro> BuscarTitulo(string busca)
         {
-            var livrosFiltrados = new List<Livro>();
-            foreach (var livro in Livros)
-            {
-                if (livro.Titulo.ToUpper().Contains(busca)) 
-                {
-                    livrosFiltrados.Add(livro);
-                }
-            }
-            return livrosFiltrados;
+            return Livros
+                .Where(l => l.Titulo.ToUpper().Contains(busca))
+                .ToList();
```

Comment:
Se o termo da busca não for em uppercase funciona?

Reply:
Já vi no teste que você faz o toUpper antes de passar para a busca, quem garante que todo mundo vai fazer isso?